File: Source/Rio.Web/src/app/shared/models/account/account-simple-dto.ts
export class AccountSimpleDto{
    AccountID: number;
    AccountName: string;
    AccountNumber: number;
    AccountVerificationKey: string;
    Notes: string;
    AccountDisplayName: string;
    ShortAccountDisplayName: string;
    
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Source/Rio.Models/DataTransferObjects/Account/AccountSimpleDto.cs
namespace Rio.Models.DataTransferObjects.Account
{
    public class AccountSimpleDto
    {
        public int AccountID { get; set; }
        public int? AccountNumber { get; set; }
        public string AccountName { get; set; }
        public string Notes { get; set; }
        public string AccountDisplayName { get; set; }
        public string ShortAccountDisplayName { get; set; }
    }
}
